cmd/view-fits/main.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2015 The astrogo Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"image"
+	"image/color"
+	"image/draw"
+	"log"
+	"os"
+
+	"github.com/astrogo/fitsio"
+
+	"golang.org/x/exp/shiny/driver"
+	"golang.org/x/exp/shiny/screen"
+	"golang.org/x/mobile/event/key"
+	"golang.org/x/mobile/event/paint"
+)
+
+var (
+	hduArray []fitsio.Image
+)
+
+func init() {
```

Comment:
I wouldn't use `init()` for this.
either fold it inside `main()` or rename it (perhaps `processFiles` or something)